bundles/org.openhab.binding.hdpowerview/src/main/java/org/openhab/binding/hdpowerview/internal/HDPowerViewWebTargets.java

```diff
@@ -64,128 +59,160 @@
     private final int maintenancePeriod = 300;
     private Instant maintenanceScheduledEnd = Instant.now().minusSeconds(2 * maintenancePeriod);
 
-    private WebTarget base;
-    private WebTarget shades;
-    private WebTarget shade;
-    private WebTarget sceneActivate;
-    private WebTarget scenes;
+    private final String base;
+    private final String shades;
+    private final String sceneActivate;
+    private final String scenes;
 
     private final Gson gson = new Gson();
+    private final HttpClient httpClient;
+
+    /**
+     * private helper class for passing http url query parameters
+     *
+     * @author AndrewFG
```

Comment:
Please remove the authoring tag on nested classes. Besides you are already listed as one of the authors of this file anyway.

Reply:
Ok.